engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -777,7 +739,8 @@ public Camera getActiveCamera() {
 
     @Override
     public Camera getLightCamera() {
-        return shadowMapCamera;
+        //FIXME: remove this method
```

Comment:
Generally I simply do a full build of my mega-source workspace, with all the stable modules present. If something breaks then we broke API :-)

This tests out fine - no compile errors. You're also *generally* safe if there's no `@API` at the top, or in a package info file. It looks like there are just four usages of this method elsewhere in the rendering system.

There are better ways to check for API violations in a more sane and thorough way. I documented one suggestion in #2317